tfbpshiny/utils/plot_formatter.py

```diff
@@ -0,0 +1,100 @@
+"""Utility functions for consistent plot formatting across the application."""
```

Comment:
This isn't applied anywhere and strikes me as a lot of AI. Try to keep the pull requests focused and small.